src/interfaces/IBullaInvoice.sol

```diff
@@ -8,25 +8,27 @@ import "../libraries/CompoundInterestLib.sol";
 struct Invoice {
     uint256 claimAmount;
     uint256 paidAmount;
+    uint256 dueBy;
+    address creditor;
+    address debtor;
+    address token;
     Status status;
     ClaimBinding binding;
     bool payerReceivesClaimOnPayment;
-    address debtor;
-    address token;
-    uint256 dueBy;
     PurchaseOrderState purchaseOrder;
     InterestConfig lateFeeConfig;
     InterestComputationState interestComputationState;
 }
 
 struct PurchaseOrderState {
     uint256 deliveryDate;
-    bool isDelivered;
     uint256 depositAmount;
+    bool isDelivered;
 }
 
+// Directionality is determined by the function that calls it
 struct CreateInvoiceParams {
-    address debtor;
+    address recipient;
```

Comment:
not sure I understand why this change is needed - we could keep it debtor, and determine if the caller is the debtor instead of having 2 functions just like in frendlend